README.md

```
@@ -78,9 +78,9 @@ git merge main
       </a>
     </td>
     <td align="center" width="150">
-      <a href="#">
-        <img src="/profile-pictures/placeholder.jpg" height="100" width="100" style="border-radius:50%;object-fit:cover;"/><br/>
-        <b>Name</b><br/><br/>
+      <a href="http://linkedin.com/in/ryanzhao27/">
+        <img src="/profile-pictures/ryan_zhao.jpg" height="100" width="100" style="border-radius:50%;object-fit:cover;"/><br/>
```

markdownlint-cli2 (0.20.0)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)